applications/Alma.md

```diff
@@ -0,0 +1,246 @@
+    - Led the engineering team at a rapidly growing startup, overseeing technical direction, team development, and product delivery.
+    - Successfully managed the development of a platform utilizing deposit-backed stablecoins, decentralized identity (DID), and verifiable credentials (VC).
+
+- **UEDA Daiki**
+  - **Education and Early Achievements**  
+    - Graduated with a Master’s degree in Computer Science from **The University of Tokyo**, focusing on advanced topics in computing.
+  - **Image Processing Product Developer**
+    - Developed image processing products, contributing to innovative solutions in the field of digital imaging.
+  - **DID/VC Product Developer**
+    - Currently working on the development of **Decentralized Identity (DID)** and **Verifiable Credentials (VC)** products, driving forward the next generation of digital identity solutions.
+
+- **ABE Motohiko**
+  - **kataru Inc. (2024)**  
+    - Specialized in Bitcoin/Lightning Network.
+  - **Questry Co., Ltd. (2022-)**
+    - Focused on EVM chains.
+  - **FreakOut Holdings, Inc. (2018-2022)**
+    - Worked on ad:tech technologies.
+  - **SORAMITSU Co., Ltd. (Part-time) (2017-2018)**
+    - Worked on private chain projects, specifically Hyperledger Iroha.
+
+
+### Team Code Repos
+
+- [https://github.com/AlmaNetwork/Alma](https://github.com/AlmaNetwork/Alma)
+
+#### Team Member
+- [https://github.com/MizukiSonoko](https://github.com/MizukiSonoko)
+- [https://github.com/yutonano](https://github.com/yutonano)
+- [https://github.com/uedadaiki](https://github.com/uedadaiki)
+- [https://github.com/takagiy](https://github.com/takagiy)
+- [https://github.com/motxx](https://github.com/motxx)
+
+
+## Development Status :open_book:
+
+We are now developing towards the release of the beta version, and we are continuously looking for new team members.
+
+
+## Development Roadmap :nut_and_bolt:
+
+### Phase 1: Initial Development and Community Engagement
+Develop core features including Identifiers, Credentials, and Secret Sharing.
+Implement ALMA region and asset management capabilities.
+
+### Phase 2: Public Beta and Community Building
+Launch a public beta version to a wider audience to test scalability and usability.
+Develop comprehensive documentation and tutorials to assist new users and developers.
+
+### Phase 3: Full Deployment and Widespread Adoption
+Roll out the full version of ALMA with all features fully operational.
+Establish a governance model to ensure sustainable development and community-driven decision making.
+
+### Phase 4: Continuous Improvement and Future Innovations
+Explore new technologies and features to keep ALMA at the forefront of digital identity and asset management.
+Foster a global network of users and developers continuously contributing to the ALMA ecosystem.
+
+### Overview
+
+- **Total Estimated Duration:** 10 month
+- **Full-Time Equivalent (FTE):**  8
+- **Total Costs:** 8,000USD
+- **DOT %:** None
```

Comment:
@MizukiSonoko to answer your question, you would basically just update this section to say 50% instead of "none". Total costs can remain the same. Also be sure to include both your USDC Asset Hub address (currently it is an ETH address) and DOT address if it is different than the USDC one. Let us know if you have questions about this.